server/routes/pt.js

```diff
@@ -114,4 +116,184 @@ router.post('/search', verifyToken, async (req, res) => {
   }
 });
 
+router.post('/addptrequest', verifyToken, async (req, res) => {
+  try {
+    const pt_id = req.body.pt_id; // ID of PT that was requested
+    const client_id = req.user.userId; // ID of client that made request
+ 
+    Pt.findByIdAndUpdate(pt_id,
+      {$push: {requests: client_id}},
+      {safe: true, upsert: true},
+      function(err,doc) {
+        if(err){
+          console.log(err);
+        }else{
+          return res.status(OK).send(client_id);
```

Comment:
```suggestion
          return res.status(OK).send();
```
No need to send `client_id`.